src/views/Header.js

```diff
@@ -34,9 +34,9 @@ type SubViewProps = NavigationSceneRendererProps & {
   onNavigateBack?: () => void,
 };
 
-type Navigation = NavigationScreenProp<NavigationState, NavigationAction>;
+type Navigation = NavigationScreenProp<*, NavigationAction>;
```

Comment:
Not sure using `*` is the correct approach here. Looking at the code below, it looks like this should probably be `NavigationScreenProp<NavigationRoute, NavigationAction>`